src/search.ts

```diff
@@ -170,13 +182,13 @@ export class Search {
 
                 let label = new St.Label({
                     text: title,
-                    styleClass: "pop-shell-search-label",
+                    styleClass: "pop-shell-search-label list-search-result-title",
                     y_align: Clutter.ActorAlign.CENTER
                 });
 
                 label.clutter_text.set_ellipsize(Pango.EllipsizeMode.END);
 
-                let container = new widgets.Box({ styleClass: "pop-shell-search-element" })
+                let container = new widgets.Box({ styleClass: "pop-shell-search-element button" })
```

Comment:
This probably should not be a `button` class, as it makes the list of items in the Shell launcher look very strange.

Reply:
Fixed